CONTRIBUTING.rst

```diff
@@ -66,9 +71,9 @@ You can test your changes under your local python environment by running the tes
     pytest
     # lint the code
     pylint cpplint.py
-    flake8
+    pre-commit run --all-files
```

Comment:
If the previous changes passed pre-commit, then newer ones should also pass, no?

Reply:
You can run this command at any time when you are working on modifications to see how the spell checker and linters will respond to your changes.